packages/block-editor/src/store/defaults.js

```diff
@@ -14,6 +14,7 @@ export const PREFERENCES_DEFAULTS = {
  * @property {boolean} alignWide Enable/Disable Wide/Full Alignments
  * @property {Array} availableLegacyWidgets Array of objects representing the legacy widgets available.
  * @property {boolean} imageEditing Image Editing settings set to false to disable.
+ * @property {string} [imageCrossOrigin] Optional value for requesting images with CORS, allowing them to be used in a `<canvas>` as if they were loaded from the current origin.
```

Comment:
I believe this is the type we want:

```suggestion
 * @property { '' | 'anonymous' | 'use-credentials' | undefined } [imageCrossOrigin] Optional value for requesting images with CORS, allowing them to be used in a `<canvas>` as if they were loaded from the current origin.
```